presto-main/src/main/java/io/prestosql/testing/SqlVarBinaryTestingUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.testing;
+
+import com.google.common.primitives.UnsignedBytes;
+import io.prestosql.spi.type.SqlVarbinary;
+
+import static com.google.common.base.CharMatcher.ascii;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.io.BaseEncoding.base16;
+import static java.nio.charset.StandardCharsets.US_ASCII;
+
+public final class SqlVarBinaryTestingUtil
+{
+    private SqlVarBinaryTestingUtil() {}
+
+    public static SqlVarbinary sqlVarbinaryHex(String value)
+    {
+        return new SqlVarbinary(base16().decode(value));
+    }
+
+    public static SqlVarbinary sqlVarbinary(String value)
```

Comment:
`..FromAscii`

Reply:
I'd prefer not, since it makes the tests harder to read. Since all of the inputs are ASCII, it should be obvious what's happening (not creating UTF-16 binary or something weird), and you get an error if you use non-ASCII, so it's safe.